platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Tools/HTMLParser.swift

```diff
@@ -37,8 +37,11 @@ final class HTMLParser {
                       encoding: String.Encoding = .utf16,
                       textColor: UIColor,
                       linkColor: UIColor,
-                      codeBackgroundColor: UIColor) throws -> NSAttributedString {
-        let htmlWithStyle = generateHtmlBodyWithStyle(htmlFragment: html, codeBackgroundColorHex: codeBackgroundColor.toHexString())
+                      codeBackgroundColor: UIColor,
+                      quoteBackgroundColor: UIColor) throws -> NSAttributedString {
+        let htmlWithStyle = generateHtmlBodyWithStyle(htmlFragment: html,
+                                                      codeBackgroundColorHex: codeBackgroundColor.toHexString(),
+                                                      quoteBackgroundColorHex: quoteBackgroundColor.toHexString())
         let attributed = try NSAttributedString(html: htmlWithStyle)
             .changeColor(to: textColor, linkColor: linkColor, codeBackgroundColor: codeBackgroundColor)
```

Comment:
i remember I have experienced some issues when creating a new line in an inline code block for the background color, so I had to force the change also in the attributed string, is everything alright for quotes?